content/blog/esc-sdk-launch/index.md

```diff
@@ -0,0 +1,226 @@
+---
+title: "Introducing Pulumi ESC SDK: Streamline Your Application Secrets Management"
+allow_long_title: true
+date: 2024-06-05T00:00:00-04:00
+draft: false
+meta_desc: "The new Pulumi ESC SDK allows developers to seamlessly integrate Pulumi ESC with their applications for secrets management"
+meta_image: "meta.png"
+authors:
+  - arun-loganathan
+  - cleve-littlefield
+tags:
+  - esc
+  - secrets
+  - features
+---
+
+Managing secrets and application configurations effectively is crucial for building secure and maintainable software. However, developers often face challenges such as hardcoded credentials, configuration inconsistencies, and security risks. [Pulumi Environments Secrets and Configuration](/product/esc) (ESC) simplifies the management of sensitive data and configuration across your entire application lifecycle. Today, we're thrilled to introduce the official Pulumi ESC SDK in [Typescript/Javascript](/docs/esc/sdk/javascript/), [Go](/docs/esc/sdk/go/), and [Python](/docs/esc/sdk/python/), making it even easier to integrate ESC directly into your applications.
```

Comment:
Note, proper capitalization is TypeScript and JavaScript (note: Script is capitalized).

Reply:
I'll update! Thank you!